saos-batch/src/test/java/pl/edu/icm/saos/batch/core/enrichment/TagPostUploadProcessingJobTest.java

```diff
@@ -48,90 +60,127 @@
     @Autowired
     private EnrichmentTagRepository enrichmentTagRepository;
     
+    @Autowired
+    private JudgmentEnrichmentService judgmentEnrichmentService;
+    
+    @Autowired
+    private JudgmentRepository judgmentRepository;
+    
+    @Autowired
+    @Qualifier("solrJudgmentsServer")
+    private SolrServer solrJudgmentsServer;
+    
     
     //------------------------ TESTS --------------------------
     
     @Test
     public void tagPostUploadJob() throws Exception {
+        
+        // given
+        
         TestObjectContext testObjectContext = testPersistenceObjectFactory.createTestObjectContext();
         
-        EnrichmentTag tag1 = testPersistenceObjectFactory.createReferencedCourtCasesTag(testObjectContext.getScJudgmentId(), testObjectContext.getNacJudgment());
-        EnrichmentTag tag2 = TestInMemoryEnrichmentTagFactory.createEnrichmentTag(testObjectContext.getScJudgmentId(), "SOME_TAG_TYPE", "{key:'value'}");
+        EnrichmentTag scjReferenceTag = createReferencedCourtCasesTag(testObjectContext.getScJudgmentId(), testObjectContext.getNacJudgment());
+        EnrichmentTag scjSomeTag = createEnrichmentTag(testObjectContext.getScJudgmentId(), "SOME_TAG_TYPE", "{key:'value'}");
+        EnrichmentTag scjMaxRefMoneyTag = createEnrichmentTag(testObjectContext.getScJudgmentId(), EnrichmentTagTypes.MAX_REFERENCED_MONEY,
+                JsonNormalizer.normalizeJson("{amount:12300.45, text:'123 tys zł 45 gr'}"));
+        
+        enrichmentTagRepository.save(Lists.newArrayList(scjReferenceTag, scjSomeTag, scjMaxRefMoneyTag));
         
-        enrichmentTagRepository.save(tag2);
+        String scJudgmentHash = getHashForTags(scjReferenceTag, scjSomeTag, scjMaxRefMoneyTag);
+        String nacJudgmentHash = getHashForTags(scjReferenceTag);
         
-        String scJudgmentHash = getHashForTags(tag1, tag2);
-        String nacJudgmentHash = getHashForTags(tag1);
         
+        // execute
         
         JobExecution execution = jobExecutor.forceStartNewJob(tagPostUploadJob);
+        solrJudgmentsServer.commit();
         
         
-        JobExecutionAssertUtils.assertJobExecution(execution, 0, 4);
+        // assert
         
-        JudgmentEnrichmentHash hash1 = judgmentEnrichmentHashRepository.findByJudgmentId(testObjectContext.getScJudgmentId());
-        assertTrue(hash1.isProcessed());
-        assertEquals(scJudgmentHash, hash1.getHash());
+        JobExecutionAssertUtils.assertStepExecution(execution, 1, 0, 4);
+        JobExecutionAssertUtils.assertStepExecution(execution, 2, 0, 2);
```

Comment:
Could you set and use some constants here instead of step numbers? For example, UPDATE_ENRICHMENT_HASH_STEP = 1.